composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "ubmannheim/publist4ubma2",
```

Comment:
Is there a reason why we should use a short name? Otherwise I'd prefer a longer name like `uni-mannheim`.